config.go

```diff
@@ -383,6 +383,8 @@ type Config struct {
 	// registered regardless of whether the RPC is called or not.
 	RequireInterceptor bool `long:"requireinterceptor" description:"Whether to always intercept HTLCs, even if no stream is attached"`
 
+	HTLCInterceptorChannelFilter uint64 `long:"htlc-interceptor-channel-filter" description:"Only intercept htlcs for the specified outgoing channel"`
```

Comment:
Could this instead be a list? Or are applications instead using a virtual scid for many channels such that this single uint64 is enough?

Reply:
I am not sure honestly. For my application, a single channel is sufficient. But I don't know about others. Maybe it could be a way to keep certain types of traffic apart. Making it a list is fairly cheap though, so I could do that.